src/kubernetes.cc

```diff
@@ -88,10 +88,21 @@ bool ReadServiceAccountSecret(
 KubernetesReader::KubernetesReader(const MetadataAgentConfiguration& config)
     : config_(config), environment_(config) {}
 
+json::value KubernetesReader::InstanceResource() const {
+  const std::string resource_type = "gce_instance";  // TODO: detect other resources.
```

Comment:
Only minor comment, does the comment need to wrap lines?

Reply:
Technically it does, but it's a TODO that will be removed, and there are other lines in this file that are over 80 characters, so maybe overkill? TODOs that aren't line comments tend to get dissociated from the appropriate lines over time.